advocacy_docs/partner_docs/PreciselyConnectCDC/04-Configuratingpreciselyconnectcdc.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: 'Configuration'
+description: 'Walkthrough on configuring the integration'
+---
+
+## Prerequisites
+
+Implementing Precisely Connect CDC with EDB Postgres Advanced Server requires the following components:
+
+- Running EDB Postgres Advanced Server instances
+- Precisely Connect CDC
```

Comment:
If we add ""Precisely Connect CDC installed and configured per your environment"?" then there will be no need to add the components and it will cause contusion.
I think the current sentence is ok but if you feel we can change it then can you please suggest the text.